CMakeSettings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "configurations": [
+  ]
+}
```

Comment:
Is this file mandatory? I see that this is related to MSVC's cmake support, but some blog about that topic mentions "If your CMake project requires additional settings...". And right now, this file seems quite empty.

BTW, a newline is missing from the end of the last line.

Reply:
OK, Indeed, this file comes from other people, I may remove it:)

Reply:
Removed